tests/test_language_detect.py

```diff
@@ -0,0 +1,79 @@
+from parameterized import parameterized, param
+from datetime import datetime
+
+from unittest.mock import Mock
+
+import unittest
```

Comment:
Minor, but it would be better to organize imports according to pep-8 https://www.python.org/dev/peps/pep-0008/#imports - so ``parameterized`` would go into a second group as it's not an stdlib module, and there would be no blank lines between stdlib modules.